packages/api/cms-api/src/content-generation/content-generation.module.ts

```diff
@@ -1,20 +1,23 @@
 import { DynamicModule, Global, Module, ModuleMetadata, Type } from "@nestjs/common";
 
-import { AzureOpenAiContentGenerationService } from "./azure-open-ai-content-generation.service";
-import { CONTENT_GENERATION_SERVICE } from "./content-generation.constants";
+import { AzureOpenAiContentGenerationService, AzureOpenAiContentGenerationServiceConfig } from "./azure-open-ai-content-generation.service";
+import { AZURE_OPEN_AI_CONTENT_GENERATION_SERVICE_CONFIG, CONTENT_GENERATION_SERVICE } from "./content-generation.constants";
 import { ContentGenerationServiceInterface } from "./content-generation-service.interface";
 import { GenerateAltTextResolver } from "./generate-alt-text.resolver";
 import { GenerateImageTitleResolver } from "./generate-image-title.resolver";
 
 export interface ContentGenerationModuleOptions {
     Service: Type<ContentGenerationServiceInterface>;
+    config: {
+        openAiContentGenerationService?: AzureOpenAiContentGenerationServiceConfig;
```

Comment:
I'd remove the Service in the config: `openAiContentGeneration?: AzureOpenAiContentGenerationServiceConfig;`

Reply:
c5e19c65b42cd459ae7e52d54d28f1278b4bf7d0